src/main/resources/templates/style.css

```diff
@@ -0,0 +1,2 @@
+body{font-family: Consolas,serif;}
+.panel{ width: 400px; height: 400px; border: solid black 1px}
```

Comment:
External CSS is going to be better for this project in the long run. As stated before, your CSS will grow as your site gets bigger. Keeping it external is better most of the time.

Reply:
https://www.hostinger.com/tutorials/difference-between-inline-external-and-internal-css

Here's a link explaining the pros and cons of all types of CSS.